src/ui/state/manager.rs

```diff
@@ -211,7 +252,10 @@ impl State for ManagerHomeState {
             Command::perform(get_blockheight(self.revaultd.clone()), Message::BlockHeight),
             Command::perform(list_vaults(self.revaultd.clone(), None), Message::Vaults),
             Command::perform(
-                list_spend_txs(self.revaultd.clone()),
+                list_spend_txs(
+                    self.revaultd.clone(),
+                    Some(&[model::SpendTxStatus::NonFinal]),
```

Comment:
I think we should also display `immature` spend tx

Reply:
I don't think we should display them here. We should do something a bit more fancy and match them with the Unvaulting we see (otherwise we have the same info duplicated!), at least for the participants that have enough information. Anyway, I would leave this for another PR :)

Reply:
I thought the immature/pending spend txs are non broadcasted spend tx, if they are not displayed how the user will broadcast them ? 
What if a user update a spend tx with a new signature, the status passed to "pending" and suddenly his laptop has no battery, how can he do the final step: broadcast the transaction ?